index.html

```diff
@@ -0,0 +1,119 @@
+        <h2 class="heading2">Registration for 2023 is Open<span> (May16) ~ (May21)</span></h2>
+        <p class="home-footer">@Web school Ethiopia, Addis Ababa</p>
+      </div>
+  </section>
+  <section class="main-program">
+    <h2 class="program-heading">Our Teching Methods</h2>
+    <div class="program-under"></div>
+    <div class="method-list">
+      <div class="method-container">
+        <div class="method-item">
+          <i class="fa-solid fa-person-chalkboard"></i>
+          <h3 class="method-item-title">Lecture</h3>
+          <p class="method-item-disc">We provide a live lecture classes by the best teachers in the topic.</p>
+        </div>
+        <div class="method-item">
+          <i class="fa-solid fa-bars-progress"></i>
+          <h3 class="method-item-title">Projects</h3>
+          <p class="method-item-disc">During the program there are lots of real world projets for practical purposes.</p>
+        </div>
+        <div class="method-item">
+          <i class="fa-solid fa-user-group"></i>
+          <h3 class="method-item-title">Pair Coding</h3>
+          <p class="method-item-disc">We will pair you with another student to stay motivated and also to learn from each other.</p>
+        </div>
+      </div>
+      <div class="method-container">
+        <div class="method-item">
+          <i class="fa-solid fa-code"></i>
+          <h3 class="method-item-title">Code reviews</h3>
+          <p class="method-item-disc">Every project you submit will be reviewed senior developers inorder to make you sharp.</p>
+        </div>
+        <div class="method-item">
+          <i class="fa-solid fa-people-pulling"></i>
+          <h3 class="method-item-title">Mentor</h3>
+          <p class="method-item-disc">In Every steps you are about to take we will make sure you get the most appropriate consling.</p>
+        </div>
+      </div>
+    </div>
+    <div class="program-btn-container"><button class="program-btn">Register Now</button></div>
+    <div class="program-disc-container"><a href="#" class="program-disc">See The Whole Program</a></div>
+    
+  </section>
+  <section class="teachers">
+
+  </section>
+  <section class="footer">
+    <div class="partners">
+      <h2 class="partners-title">Partners</h2>
+      <div class="partners-under"></div>
+      <div class="partners-list">
+        <div class="partners-item">
+          <a href="#" class="mozilla">mozilla</a>
+          <a href="#" class="google">Google</a>
+          <a href="#" class="naver">NAVER</a>
+          <a href="#" class="daunmkakao">daumkakao</a>
+          <a href="#" class="airbnb">airbnb</a>
+        </div>
+      </div>
+    </div>
+  </section>
+  <div class="copyright">
+    <h2 class="copy-title">Web School</h2>
+    <p class="copy-disc">2023 Web School Ethiopia, some Rights Reserved.</p>
+  </div>
```

Comment:
- [ ] Kindly wrap your footer into `footer` tag. That would be semantically correct.
- [ ] Kindly align your footer text to the center to match the given design:
![image](https://user-images.githubusercontent.com/91486791/235132328-66d72700-a100-456e-9055-c012ccc37d38.png)
![image](https://user-images.githubusercontent.com/91486791/235132390-a8853bac-9709-4a45-9185-81dfa0aa79e9.png)